src/event.lib.php

```diff
@@ -819,7 +822,7 @@ public static function GetPasswordResetKey($user)
         );
 
         if (is_wp_error($key_saved)) {
-            return false;
+            return $key_error;
```

Comment:
The [`get_password_reset_key`](https://core.trac.wordpress.org/browser/tags/5.6/src/wp-includes/user.php#L2559) function returns the `$key_saved` variable on error.

Reply:
Great catch!